sdv/logging/logger.py

```diff
@@ -1,11 +1,35 @@
 """SDV Logger."""
-
+import csv
 import logging
 from functools import lru_cache
+from io import StringIO
 
 from sdv.logging.utils import get_sdv_logger_config
 
 
+class CSVHandler(logging.Formatter):
```

Comment:
Could we use a different name for this ? I'm a bit concerned about having another class already with this name:
https://github.com/sdv-dev/SDV/blob/08a464c10d82f054e00d5807f6574579d7204329/sdv/io/local/local.py#L54

Reply:
Ah good catch, it should probably be `CSVFormatter` to be clearer.